### PR TITLE
MLP Project Landing Page

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
Update the Project Landing page to display the summary of ML project components.

image

### CODE REVIEW

File: ui/packages/app/src/pages/project/components/ComingSoonPanel.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+} from "@elastic/eui";
+
+export const ComingSoonPanel = ({ title, iconType }) => {
+  return (
+    <EuiCard
+      icon={<EuiIcon size="xl" type={iconType} />}
+      title={title}
+      description="Coming soon."
+      layout="horizontal"
+      // betaBadgeLabel={badges[index]}
```
Comment (tiopramayudi) [inline ui/packages/app/src/pages/project/components/ComingSoonPanel.js:14]: can you remove commented code?

File: ui/packages/app/src/pages/project/components/FeastJobsTable.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          if (job.type === "STREAM_INGESTION_JOB") {
+            tableName = job.streamIngestion.tableName;
+          } else if (job.type === "BATCH_INGESTION_JOB") {
+            tableName = job.batchIngestion.tableName;
+          }
+
+          items.push({
+            id: job.id,
+            table_name: tableName,
+            type: job.type === "STREAM_INGESTION_JOB" ? "stream" : "batch",
```
Comment (tiopramayudi) [inline ui/packages/app/src/pages/project/components/FeastJobsTable.js:27]: why not set jobtype in the logic above?

File: ui/packages/app/src/pages/project/components/MerlinDeploymentsTable.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        .filter(
+          model =>
+            model.endpoints &&
+            model.endpoints.length > 0 &&
+            model.endpoints.find(endpoint => endpoint.status === "serving")
+        )
+        .sort((a, b) => (a.name > b.name ? 1 : -1))
+        .forEach(model => {
+          model.endpoints.forEach(endpoint => {
+            if (items.length >= 5) {
```
Comment (tiopramayudi) [inline ui/packages/app/src/pages/project/components/MerlinDeploymentsTable.js:20]: why 5?

File: ui/packages/app/src/pages/project/components/ProjectSummary.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    {
+      title: "Created at",
+      description: moment(project.created_at).format("DD-MM-YYYY")
+    },
+    {
+      title: "Modified at",
+      description: moment(project.updated_at).format("DD-MM-YYYY")
+    },
+    // {
```
Comment (tiopramayudi) [inline ui/packages/app/src/pages/project/components/ProjectSummary.js:34]: can you remove it ?

File: ui/packages/app/src/pages/project/components/TuringRoutersTable.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    if (project && routers) {
+      let items = [];
+      routers
+        .filter(router => router.status === "deployed")
+        .sort((a, b) => (a.name > b.name ? 1 : -1))
+        .forEach(router => {
+          if (items.length >= 5) {
```
Comment (tiopramayudi) [inline ui/packages/app/src/pages/project/components/TuringRoutersTable.js:14]: why 5?

File: ui/packages/app/src/services/merlin/Merlin.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { useContext, useEffect, useState } from "react";
+import { ApplicationsContext } from "@gojek/mlp-ui";
+
+export const MerlinApp = () => {
+  const { apps } = useContext(ApplicationsContext);
+  const [merlinApp, setMerlinApp] = useState({});
+
+  useEffect(() => {
+    console.log("WKWKWKKWKKKK 1111");
```
Comment (tiopramayudi) [inline ui/packages/app/src/services/merlin/Merlin.js:9]: WKWKWKWK

File: ui/packages/app/src/pages/project/components/MerlinDeploymentsTable.js (hunk 3 above)
Comment (ariefrahmansyah) [reply ui/packages/app/src/pages/project/components/MerlinDeploymentsTable.js:20]: we don't want it to be bloated so we only limit to 5 serving model. if users want to see more, they can click view all

**Comment by tiopramayudi** [comment]
LGTm

**Pull request merged**